styleguide/Colors.js

```diff
@@ -157,42 +174,48 @@ class Colors extends React.Component {
             value={this.state.newColor}
             onChange={this.handleChange}
           />
+          <label style={{ marginLeft: '10px' }}>
+            <input type="checkbox" onChange={this.toggleBg} />
+            Toggle bg
```

Comment:
`Toggle background`? :P I don't think I'd know what `bg` refers to from the UI, but then again I'd probably just click it :)